### ISSUE TITLE
Runtime errors prevent start

### ISSUE THREAD

**Issue by appurist** [thirdweb-example/sign-in-with-ethereum#1]
I tried cloning this repo and running `yarn` and `yarn dev` after setting my Metamask private key as `ADMIN_PRIVATE_KEY` in a `.env` file. However, I see a white screen on localhost:3000 and plenty of errors in the console. The initial page does not load, see image below.

I'm thinking this may be related to versions of react vs react router, node, or some other dependency. It may be useful to commit the `yarn.lock` or `npm-lock.json` file in order to ensure compatible versions. (Or it you can just try attaching one to this issue I can try it to see if the issue goes away, then compare it with mine.)

![image](https://user-images.githubusercontent.com/1620487/177661932-614c1808-693f-4ba0-b32c-7630b65d82da.png)

**Comment by jarrodwatts** [comment]
Hey, @appurist I am not able to replicate this on my machine.

If you want to, are you able to jump in the discord (https://discord.gg/thirdweb) and ping @adam-maj and myself and see if we can work out why this is happening for you?

**Comment by appurist** [comment]
@jarrodwatts After posting this, I saw the reference to the Discord server and joined earlier today. I will ping you both there. 👍

**Comment by appurist** [comment]
I've attached the console output from `yarn dev` here.
Lots of errors about only case-sensitive module names, although it's difficult to sort out which ones it's referring to even though it seems they are named. Note: I am running this on Windows so the file system is case-insensitive. The other errors are some unexpected undefined errors in react hook calls, so it does feel like a build/environment problem of some kind.

I did try removing the `node_modules` folder and reinstalling and **did** get the same resulting `yarn.lock` that is in the repo. So it's not dependency version differences, but perhaps platform differences.

[err.txt](https://github.com/thirdweb-example/sign-in-with-ethereum/files/9068424/err.txt)

Chrome 103.0.5060.66 (Official Build) (64-bit)
Node 16.15.1
Windows 11

**Comment by appurist** [comment]
At Joakim's suggestion, I tried `npx thirdweb create` and it suffers from the same errors on a `yarn dev`.
I wonder if this is just some kind of incompatibility (perhaps Windows-specific) in more recent minor upgrades to dependencies. 
@jarrodwatts and @adam-maj if one of you gets a chance, can you try `npx thirdweb create` in a new folder? Or a `yarn upgrade` on your existing one?  I'm thinking maybe it's just that I'm trying this today and yours ran previously with slightly older dependencies? (Although I guess not, with the same `yarn.lock`... This is a weird one.)

![image](https://user-images.githubusercontent.com/1620487/177907778-93259513-5aab-4ee7-8c1e-cffb5a68ff1a.png)

**Comment by appurist** [comment]
Thanks to the help of @jarrodwatts, @adam-maj Joaquim and Devin in Discord, keeping me sane while we tried to narrow down the issue, I have resolved it. The problem appears to be a case-sensitivity issue, visible in Next but probably many other packages, and reported [here](https://github.com/vercel/next.js/issues/10739) (among other similar reports in other projects).

I believe the problem was that my path sometimes showed as `c:\dev\`... and sometimes as `C:\dev\`... and that caused several path mismatches to occur in the module resolution. And for whatever reason, that caused `null` errors in some react hooks.  It may have also been related to the way I was launching a Command Prompt from a Total Commander shell field, which lowercases the drive letters.

At any rate, after restarting my machine, and manually running `cmd` from the Windows Run... command, then using a root-relative path in a CD from there (`\dev\`...), it just worked. I still got a couple of warnings about case-sensitivity, but the app worked, and I was able to connect to my Metamask wallet and sign in with Ethereum.

Hope this helps someone in the future. I'm going to close this as it seems to have nothing to do with any thirdweb code anyway.
